test/e2e/runner/main.go

```diff
@@ -188,6 +186,23 @@ func NewCLI() *CLI {
 		},
 	})
 
+	cli.root.AddCommand(&cobra.Command{
+		Use:   "evidence [amount]",
+		Short: "Generates and broadcasts evidence to a random node",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
```

Comment:
Let's add a max args here just for safety 👍

Reply:
Yeah good catch